src/mantine-notifications/src/NotificationsProvider/use-notifications-state/use-notifications-state.ts

```diff
@@ -7,7 +8,7 @@ export default function useNotificationsState({ limit }: { limit: number }) {
     limit,
   });
 
-  const showNotification = (notification: NotificationProps) => {
+  const showNotification = useCallback((notification: NotificationProps) => {
     const id = notification.id || randomId();
 
     update((notifications) => {
```

Comment:
Why doesn't this produce a warning in ESLint? Does `eslint-plugin-react-hooks` know that `update` is okay here?